mcstatus/__main__.py

```diff
@@ -42,12 +41,18 @@ def json(server: JavaServer) -> None:
         if status_res.players.sample is not None:
             data["players"] = [{"name": player.name, "id": player.id} for player in status_res.players.sample]
 
+        #ping_res = server.ping() # status calls test_ping(), a 2nd ping is not needed.
+        #data["ping"] = ping_res
+        data["ping"] = status_res.latency
+        data["online"] = True
+
         query_res = server.query(tries=1)  # type: ignore[call-arg] # tries is supported with retry decorator
         data["host_ip"] = query_res.raw["hostip"]
         data["host_port"] = query_res.raw["hostport"]
         data["map"] = query_res.map
         data["plugins"] = query_res.software.plugins
-    except Exception:  # TODO: Check what this actually excepts
+    except Exception as e:  # TODO: Check what this actually excepts
+        #print(e)
```

Comment:
Try not leaving debug prints in commits.
```suggestion
    except Exception:  # TODO: Check what this actually excepts
```

This also leaves `e` to be an unused variable, which is an anti-pattern.